site/en/blog/run-on-login/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Automatically start PWAs on OS Login
+subhead: |
+  Installed PWAs can now be configured to run on login.
+authors:
+  - ajara
+date: 2021-07-29
+description: |
+   Many operating systems offer the ability to launch applications automatically when the user logs in. This ability is now available for installed web applications.
+hero: image/SeARmcA1EicLXagFnVOe0ou9cqK2/e5ATqM7ge5Y3Sxdqf1T8.jpg
+alt: Laptop, coffee, and stationary tools on desk.
+---
+Run on login gives the user the ability to choose which applications start
+automatically when they first log into their computer. This ability has been
+available in many operating systems for a long time, but was reserved for
+platform specific applications. Now it is available for installed web applications too.
+
+## Save users time
+
+Users want to automatically run a set of applications they always use. A
+typical example is high-engagement applications like email clients, chat apps,
+monitoring dashboards, and real-time data display apps, to name just a few.
+
+Auto-starting these apps at login streamlines the user experience by avoiding
+having to find these apps to launch in menus.
```

Comment:
```suggestion
Auto-starting these apps at login streamlines the user experience by saving
users from having to find these apps to launch in menus.
```

Reply:
Grammatically, the two parts of the sentence are structured as though they refer to the same actor. This change corrects that.